src/lib/okta.connect.ts

```diff
@@ -0,0 +1,12 @@
+import OktaJwtVerifier from "@okta/jwt-verifier";
+
+const { OKTA_URI = "playtix.okta.com", OAUTH_SERVER = "default" } = process.env;
```

Comment:
let's create a `config.ts` file next to `index.ts` that is responsible for reading environent variables, setting the default and validating them. Similarily to what we do here: https://github.com/jwa-lab/tokenization-service/blob/main/src/config.ts

We should this line, and the equivalent in `index.ts` to that `config.ts` file, and we would just import those variables here

Reply:
more importantly, when we run in production mode, the OKTA_URI should be the production URI (playtix.okta.com), but when we run in development, it should be a dev url. The url should be `localhost` and we should have a mock "okta server" running locally that just returns "ok" on that localhost url.

To switch between prod and dev URL, we would simply set the environment variable to the proper value when we start airlock. So on the local machine it the env variable can come from our terminal and injected into the docker container, and when it will run in the cloud it will be set by our deployment scripts.

Reply:
one last thing, let's not set a default for critical environment variables, let's just exit the process with an error if the environment variable isn't set.
It's too important that this environment variable is set to the right value (like not bypassing auth in production) that if the env variable isn't set the process should crash, instead of starting with the wrong url without the dev team noticing